### PR TITLE
docs: add instructions to make a documentation page

### PR DESCRIPTION
**Pull request by IshitaTakeshi**
## Description

Add instructions to make a documentation page

https://github.com/autowarefoundation/autoware-documentation/issues/84

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [ ] I've confirmed the [contribution guidelines].
- [ ] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The Reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by kenji-miyake** [comment]
@IshitaTakeshi Please read my comments in https://github.com/autowarefoundation/autoware-documentation/issues/84 carefully. :pray: 

And please follow https://autowarefoundation.github.io/autoware-documentation/main/contributing/.
https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/#ci-checks would be helpful for you.

**Comment by IshitaTakeshi** [comment]
I read it. What's the problem now

File: docs/contributing/documentation-guidelines/index.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Documentation guidelines
 
-!!! warning
+## Instructions to add documentation
 
-    Under Construction
+There are two ways to contribute to the documentation page.
```
Comment (kenji-miyake) [inline docs/contributing/documentation-guidelines/index.md:5]: This page isn't for how to contribute, but how to write good documents.
So you should edit https://autowarefoundation.github.io/autoware-documentation/main/contributing/#contribution-workflows or https://autowarefoundation.github.io/autoware-documentation/main/help/support-guidelines/#github-discussions instead.

File: docs/contributing/documentation-guidelines/index.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
...
+
+1. If you can make a documentation page based on your knowledge, please [send a pull request](https://github.com/autowarefoundation/autoware-documentation/pulls). If you are unaware of if you can open a pull request or unaware of the format, you can ask on the [feature requests page](https://github.com/autowarefoundation/autoware/discussions/categories/feature-requests).
```
Comment (kenji-miyake) [inline docs/contributing/documentation-guidelines/index.md:7]: The feature request category isn't for documentation.
Please use Q&A instead.

**Review by kenji-miyake — CHANGES_REQUESTED** [review]

File: docs/contributing/documentation-guidelines/index.md (hunk 1 above)
Comment (IshitaTakeshi) [reply docs/contributing/documentation-guidelines/index.md:5]: That's what I wanted to know

**Comment by kenji-miyake** [comment]
@IshitaTakeshi Could you update this PR? If it's resolved by #91, please close #84. :bow:

**Comment by IshitaTakeshi** [comment]
Closed this because of updates in https://github.com/autowarefoundation/autoware-documentation/pull/91

**Pull request closed without merging**